#  Message describing a set of stamped trajectories

Vec3GroupTrajectory[] trajectory_samples
